src/installer/managed/Microsoft.NET.HostModel/Bundle/TargetInfo.cs

```diff
@@ -79,7 +79,7 @@ public TargetInfo(OSPlatform? os, Architecture? arch, Version targetFrameworkVer
 
         public bool IsNativeBinary(string filePath)
         {
-            return IsLinux ? ElfUtils.IsElfImage(filePath) : IsOSX ? MachOUtils.IsMachOImage(filePath) : PEUtils.IsPEImage(filePath);
+            return IsLinux || IsFreeBSD ? ElfUtils.IsElfImage(filePath) : IsOSX ? MachOUtils.IsMachOImage(filePath) : PEUtils.IsPEImage(filePath);
```

Comment:
```suggestion
            return IsWindows ? PEUtils.IsPEImage(filePath) : IsOSX ? MachOUtils.IsMachOImage(filePath) : ElfUtils.IsElfImage(filePath);
```
We can use ELF is the fallback options. It is used by number of Unix systems.